test/unit/connectors/CustomsDataStoreConnectorSpec.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import base.UnitSpec
+import config.AppConfig
+import org.scalatest.GivenWhenThen
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.test.Injecting
+import testdata.TestData
+
+class CustomsDataStoreConnectorSpec extends UnitSpec with GivenWhenThen with GuiceOneAppPerSuite with Injecting {
+
+  implicit val appConfig: AppConfig = inject[AppConfig]
+
+  "CustomsDataStoreConnector.url" should {
+    "correctly compose the url to verify and retrieve the email address" in {
```

Comment:
I'd also add some tests for successful/failed calls. To document the behavior of the connector this way.

Reply:
But here you would test the library, and we do not want to do this. There is no code in the connector.get()

Reply:
Added a test for this, but tbh I do not see any value/safety in having this test.